.editorconfig

```diff
@@ -0,0 +1,16 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[Makefile]
+indent_style = tab
```

Comment:
@matheuslc do you have the editorconfig extensions installed? It should have added a line at the end of the file 👀

Reply:
I thought I had, I check and it was already installed 🤔 But I had to save again each file to add the missing new line at the end of the file. Thanks! 🙏 Fixed ✅